stdlib/websub/src/main/ballerina/src/websub/hub_service.bal

```diff
@@ -385,11 +385,11 @@ function verifyIntentAndAddSubscription(string callback, string topic, map<strin
     }
 }
 
-# Function to add/remove the persisted details of topics registered.
+# Adds/Removes the persisted details to/from the topics registered.
 #
 # + mode - Whether the change is for addition/removal
 # + topic - The topic for which registration is changing
-# + return - `error` if an error occurred while persisting the change, `()` otherwise
+# + return - An `error` if an error occurred while persisting the change or else `()`
```

Comment:
```suggestion
# + return - An `error` if an error occurred while persisting the change or else `()`
```
Need to specify the error.

Reply:
We are using a common error in this module. So we need to specify the error.

Reply:
+1